core/cas-server-core-services/src/main/java/org/apereo/cas/services/AbstractResourceBasedServiceRegistryDao.java

```diff
@@ -48,12 +51,16 @@
         LOG_SERVICE_DUPLICATE.accept(s2);
         return s1;
     };
+    private static final int INTERVAL = 0;
 
     /**
      * The Service registry directory.
      */
     protected Path serviceRegistryDirectory;
 
+    // PLEASE REVIEW: make sense to this publisher to live in a where ApplicationEventPublisher lives?? and just reference to it
+    private Consumer<AbstractCasEvent> publish = this::publishEvent;
```

Comment:
I am not sure I follow your comment here. Could you explain please. Are you saying the publishEvent() call should turn into a separate EventPublisher component?